### ISSUE TITLE
Requiring react 0.14.0?

### ISSUE THREAD

**Issue by nicholasalanbrown** [FormidableLabs/react-shuffle#17]
I can't get this to work with Browserify since you have a peer dependency on react v0.14.0, which as far as I can tell from the React GitHub repo, isn't out yet (current release is 0.13.3). Is there any reason you're requiring a version higher than the latest stable release?

Real shame, I'd love to use this in my app!

**Comment by KeKs0r** [comment]
Actually react 0.14 is already released.
Also this packages has also older versions:
`[ '0.0.1', '0.0.2', '0.0.3', '0.1.0', '0.1.1', '1.0.0' ]`
So just try one before 1.0.0 since I think that version introduced the react 0.14 dependency

**Comment by coreylenertz** [comment]
Looks like it's time to support React 15 as well

**Comment by pencilcheck** [comment]
Yup, in peerDependency it listed react ^15 but it doesn't really work with react 15
